app/options.go

```diff
@@ -69,6 +73,12 @@ func StartElasticsearch(s *Server) error {
 	return nil
 }
 
+func SetLogProxy(tb testing.TB) Option {
```

Comment:
This method should probably be named `SetLogger` and accept just a `*mlog.Logger`. Then the calling code can leverage `NewLogProxy`, but the option becomes more generally useful for future code that might want to override the logging for other reasons.